evalml/data_checks/datetime_nan_data_check.py

```diff
@@ -29,7 +29,9 @@ def validate(self, X, y=None):
             ...         ['2-4-21', '3-4-21']]
             >>> df = pd.DataFrame(dates, columns=['index', "days"])
             >>> dt_nan_dc = DateTimeNaNDataCheck()
-            >>> assert dt_nan_dc.validate(df) == {'warnings': [], 'errors': [], 'actions': []}
+            >>> assert dt_nan_dc.validate(df) == {"warnings": [], "errors": [], "actions": {"action_list": [], "default_action": None}}
+            ...
+            ...
```

Comment:
Are these necessary? I'm not sure I understand the details of doctest syntax.

Reply:
Don't think so, removing!